src/defines.h

```diff
@@ -59,9 +59,14 @@
 //   BAT_EXTERNAL for ADC pin,
 //   BAT_INTERNAL for internal - can detect only low battery,
 //   BAT_MCP3021 for external ADC connected over I2C
+//   BAT_BQ27441 for external BQ27441 battery gauge IC
 #define BATTERY_MONITOR BAT_EXTERNAL
 #endif
 
+#if BATTERY_MONITOR == BAT_BQ27441 && I2C_SPEED > 100000
+#error BQ27441 library supports only 100 kHz I2C speed
+#endif
```

Comment:
Not sure how well this requirement would work, the sensor reading speed is already somewhat the bottleneck with 400kHz

Reply:
You're right, for daily use I use trackers iwth single IMU but if I tried it with 2nd one it got laggy.
I improved the sparkfun lib to support i2c >100 kHz.